input/docs/getting-started/installation/nuget-packages/index.md

```diff
@@ -15,23 +15,27 @@ Order: 10
 Assuming the following project structure:
 
 ```
-MyCoolApp (application)
-MyCoolApps.UnitTests (tests)
+- MyCoolApp (netstandard)
+- MyCoolApp.WinForms (application)
+- MyCoolApp.UnitTests (tests)
 ```
 
+* Install `ReactiveUI` into your netstandard library, application and tests.
 * Install `ReactiveUI.WinForms` into your application and tests.
-* Install `ReactiveUI.Events.WinForms` into your application (NOTE: at time of writing, this package is missing from NuGet).
+* Install `ReactiveUI.Events.WinForms` into your application.
 * Install `ReactiveUI.Testing` into your tests.
 
 # Windows Presentation Foundation
 
 Assuming the following project structure:
 
 ```
-- MyCoolApp (application)
-- MyCoolApps.UnitTests (tests)
+- MyCoolApp (netstandard)
+- MyCoolApp.WPF (application)
+- MyCoolApp.UnitTests (tests)
 ```
 
+* Install `ReactiveUI` into your netstandard library, application and tests.
```

Comment:
Winforms/WPF applications tend to have more assemblies due to the cost of assembly loading is much less on those platforms. Maybe reword it to be "Install ReactiveUI into your netstandard libraries, application and tests. I know you are attempting to explain the above project assembly structure but given it's a common use case on these platforms it might be worth explaining it?

Reply:
I agree this makes sence, thanks.